db/migrate/20231128092231_update_default_amount_details_on_fees.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class UpdateDefaultAmountDetailsOnFees < ActiveRecord::Migration[7.0]
+  class Fee < ApplicationRecord; end
+
+  def change
+    change_column_default :fees, :amount_details, from: '{}', to: {}
+
+    reversible do |dir|
+      dir.up do
+        # Update all existing fees with '{}' value to {}
+        Fee.where(amount_details: '{}').update_all(amount_details: {})
```

Comment:
Rails/SkipsModelValidations: Avoid using `update_all` because it skips validations. (https://guides.rubyonrails.org/active_record_validations.html#skipping-validations)